RELEASING.md

```diff
@@ -40,91 +18,26 @@ cd documentation
-
-```bash
-# find package name in the list 
-npm access ls-packages | grep 'reveal'
-``` 
-
-If you see `"@cognite/<package-name>": "read-write"` you're good to go. 
-If you don't have access, then you need to create a PR where you add your npm username to access file. See [this pull request](https://github.com/cognitedata/terraform-npm/pull/14/files) for example.
-
-If you are not currently logged into npm on your CLI, do the following:
-* Run `npm login` (must be `npm login`, not `yarn login`).
-* Enter your npm username and password.
-* Enable [2FA](https://docs.npmjs.com/configuring-two-factor-authentication) for your npm account. 
-    You will need to enter one time password (OTP) to publish a package.
-
-Once logged in, run:
-
-```bash
-yarn
-yarn build:prod
-cd dist
-npm publish
-```
-
-or if you are releasing a beta version:
+1. Commit these changes
 
-```bash
-yarn
-yarn build:prod
-cd dist
-npm publish --tag=beta
-```
-
-It creates a build, copies package.json into /dist with modifications and runs npm publish from viewer/dist.
+2. Create a PR with the changes.
 
-Now, if published successfully, **merge the pull request**.
-
-In case if you're releasing reveal-parser-worker, you must upload it to CDN after successful release. 
-To do that, go to GitHub Actions and run [Upload worker workflow](https://github.com/cognitedata/reveal/actions/workflows/upload-worker.yml)
-targeting the master branch.
+Once the PR is merged into master, the documentation will be automatically deployed.
 
 ### Create a release on GitHub
 
 1. Go to [https://github.com/cognitedata/reveal/releases/new](https://github.com/cognitedata/reveal/releases/new)
-1. Under the "Tag version" field, look for the newly-created tag, e.g. `@cognite/reveal@2.1.0`
-1. Specify the same release title as the tag name.
-1. Write the changes that new version brings. Get inspired by done tasks from your sprint board. Note! 
+2. Under the "Tag version" field, add a new tag for this release with the name and version of the package, e.g. `@cognite/reveal@3.2.0`
+3. Specify the same release title as the tag name.
+4. Write the changes that new version brings. Get inspired by done tasks from your sprint board. Note! 
    Remember to specify the correct ThreeJS version required by the released version - you can find this in package.json 
    of the viewer.
 Also, you can check what's committed from the previous tag with that command:
    ```bash
-   git log --pretty=format:"* %s" @cognite/reveal@2.0.0...HEAD
+   git log --pretty=format:"* %s" @cognite/reveal@3.2.0...HEAD
    ```
-   Use the following template:
-   ```md
+    Use the following template:  
+```
 Reveal comes with ThreeJS embedded so you do not have this as a dependency in your project. If you still want to have it as a direct dependency, it must match the version used by Reveal (r<THREEJS_VERSION>). You can also use three.js version exported by Reveal as import { THREE } from '@cognite/reveal.
```

Comment:
Should we remove this part - or wait until the threejs- peer dependency PR is merged?

Reply:
Let's wait for peer dependency. We may hit something that ends up messing this up.